patches/third_party-blink-renderer-core-frame-navigator.h.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/third_party/blink/renderer/core/frame/navigator.h b/third_party/blink/renderer/core/frame/navigator.h
+index 4572df017cf4..377827ad9a3c 100644
+--- a/third_party/blink/renderer/core/frame/navigator.h
++++ b/third_party/blink/renderer/core/frame/navigator.h
+@@ -24,7 +24,6 @@
+ #include "third_party/blink/renderer/core/core_export.h"
+ #include "third_party/blink/renderer/core/execution_context/context_lifecycle_observer.h"
+ #include "third_party/blink/renderer/core/frame/navigator_concurrent_hardware.h"
+-#include "third_party/blink/renderer/core/frame/navigator_device_memory.h"
```

Comment:
we are discussing exactly what we want to do here (returning a fixed value vs removing the api, but either way removing this header is not necessary and I'm not even sure if removing https://github.com/brave/brave-core/pull/3669/files#diff-231a100e06059effa4a4b129d96e35fcR17 is necessary. Is https://github.com/brave/brave-core/pull/3669/files#diff-04ad001bca876052b0b18cc9dfa3df9dR9 alone not sufficient?

Reply:
You are right, this patch is unnecessary, I removed it.